oci/layout/oci_dest.go

```diff
@@ -32,7 +33,16 @@ func newImageDestination(ref ociReference) (types.ImageDestination, error) {
 			SchemaVersion: 2,
 		},
 	}
-	return &ociImageDestination{ref: ref, index: index}, nil
+	d := &ociImageDestination{ref: ref, index: index, sharedBlobDir: ctx.OCISharedBlobDirPath}
+
+	if err := ensureDirectoryExists(d.ref.dir); err != nil {
+		return nil, err
+	}
+	// https: //github.com/opencontainers/image-spec/blame/7c889fafd04a893f5c5f50b7ab9963d5d64e5242/image-layout.md#L19
```

Comment:
Extra space after `https:`.  (Preferably, quote that single sentence in the comment as well, to save future readers’ time.)